keine cpu limits (#128)
it-at-m/helm-charts: charts/refarch-templates/README.md

        cpu: 100m
        memory: 512Mi
      limits:
        memory: 512Mi
    replicas: 1
```

> > **Note:** You can also just override the `requests` or `limits`, but always need to set `cpu` and `memory` if you do so. You don't need to specifie the CPU-Limit in the new cluster. The pod takes all the availbe cpu of a node for a multithreating process.

Reviewer: I`d remove the reference to the cluster:

```suggestion
> **Note:** You can also just override the `requests` or `limits` individually. If you do so, the defaults for both `cpu` and `memory` will not apply.
```
